gp_align/analysis.py

```diff
@@ -184,7 +207,7 @@ def analyze_image(args):
                 columns)
             assert len(well_centers) == rows * columns
 
-            plate_image /= (1 - plate_image)
+            plate_image /= (1 - plate_image + 0.001)
```

Comment:
0.001 seems quite large or does it not matter? What about using `sys.float_info.epsilon`?

Reply:
It will only make a difference for intensities very close to 1, where the sensitivity is really bad either way. The exact transformed values of these high-intensity pixels do not matter, but if they are transformed to nan they will not be included when calculating the median of the brightest n pixels in a well.